Library/Homebrew/utils/analytics.rb

```diff
@@ -3,6 +3,10 @@
 module Utils
   module Analytics
     class << self
+      def clear_anonymous_os_prefix_ci_cache
+        remove_instance_variable(:@anonymous_os_prefix_ci) if instance_variable_defined?(:@anonymous_os_prefix_ci)
```

Comment:
Can you split this onto two lines or add an early return (`return unless instance_variable_defined`...)? Thanks!